src/linker/implementation.py

```diff
@@ -11,26 +15,51 @@ def __init__(
         implementation_name: str,
         implementation_config: Optional[Dict[str, Any]],
         container_engine: str,
+        environment: Dict[str, Any],
```

Comment:
It's not super clear to me what this is.

Reply:
Oh, is it for binding in the necessary files? That wasn't super clear to me

Reply:
Yeah. Maybe there's a better name. Basically right now the only implementation configuration we support are env variables so I thought `environment` was an appropriate name?

Reply:
Maybe just `configuration` with the implicit understanding that we really mean variables?

Reply:
But this isn't implementation configuration, right? That's `implementation_config`. IIUC this is only being used for binding in files, so you might want to just call it file bindings or something.

Reply:
Sorry, I was incorrect before. This environment attribute is used for the spark and slurm requests, i.e.
```
logfile, job_id = submit_spark_cluster_job(
        drmaa=drmaa,
        session=session,
        launcher=launcher,
        step_id=step_id,
        account=environment["slurm"]["account"],
        partition=environment["slurm"]["partition"],
        memory_per_node=environment["spark"]["workers"]["mem_per_node"],
        max_runtime=environment["spark"]["workers"]["time_limit"],
        num_workers=environment["spark"]["workers"]["num_workers"],
        cpus_per_node=environment["spark"]["workers"]["cpus_per_node"],
        preserve_logs=preserve_logs,
    )
```

Perhaps just `resource_requests`?

Reply:
Ahh, I understand. Maybe `computing_environment_config` or something.

Reply:
Actually, I'm still wrong. This is the `environment.yaml` dictionary. Hence the name `environment`

Reply:
Yeah, I understood that. That's why I suggested `computing_environment_config` since it is a bit less ambiguous. But, don't let me hold you up from working on actual things!

Reply:
Ok, I think it's better? Called it `resources` but also stripped out the actual resources from the environment config.